packages/utils/src/range.ts

```diff
@@ -1,11 +1,17 @@
 /**
- * Generate an array of numbers from start to stop with a step
+ * The `range` function generates an array of numbers from a specified starting
+ * point to a stopping point, with a defined step size. It can create both
+ * inclusive and exclusive ranges based on the provided parameters. This
+ * function is useful for generating sequences of numbers for loops,
+ * iterations, or other numerical operations.
  *
- * @param start - The start of the range
- * @param stop - The end of the range
- * @param step - The step between each number
- * @param inclusive - Include the stop number in the range
- * @returns An array of numbers
+ * @param start - The starting value of the range.
+ * @param stop - The ending value of the range.
+ * @param step - The increment between each number in the generated range. Can
+ * be a positive or negative number.
+ * @param inclusive - If `true`, the stopping value will be included in the
+ * generated array. If `false`, the stopping value will be excluded.
+ * @returns An array of numbers representing the specified range.
```

Comment:
**todo:** keep the same terminology as in the description of the function.

Either replace `stopping point` with `value` or vice versa.